app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -176,3 +178,22 @@ export default {
   },
 };
 </script>
+
+<style lang="scss">
+
+.export-submissions-dlg {
+  // vuetify radio button labels need alignment help
+  .v-radio {
+    > label {
+      margin-bottom: 0;
+    }
+  }
```

Comment:
Left this comment on Tim's PR #69 looks like he's adjusting it there too so would conflict with this PR. Should scope higher than the export dialog anyways.

_If we're going to adjust the radio button vertical alignment issue it should be a global style setting. It's not just this export dialog with the issue
See "Vertical alignment of text for Checkboxes and Radio buttons (see Create New Form) too high." in https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1656

This is (another) issue introduced by Bootstrap it looks like. So if we do want to address it quickly with CSS overrides can apply a class to places it needs to be fixed globablly in style.scss maybe rather than scoped.

We should double check the radios and checkboxes on the actual FormIO forms themselves too._

Reply:
yup that can come out when we get a global solution.

Reply:
One of you or @TimCsaky will have to take out your implementation though since you're both doing it in pr 68 and 69 in different ways. (or both take it out and move to style.scss)

Reply:
left a comment in @TimCsaky PR that this has moved globally.